.github/workflows/mlc_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ignorePatterns": [
+      {
+        "pattern": "^https://github.com/"
+      },
+      {
+        "pattern": "^https://docs.github.com/"
+      }
+    ]
+}
```

Comment:
Why this `ignorePatterns`? Maybe better to fix the issue this way https://github.com/gaurav-nelson/github-action-markdown-link-check#github-links-failure-fix

Reply:
I do not remember why I did it this way 🤔

Reply:
It was github rate limiter

but it's working this time 🤷